.github/workflows/linter.yml

```diff
@@ -47,3 +47,4 @@ jobs:
           VALIDATE_ALL_CODEBASE: true
           VALIDATE_JAVASCRIPT_STANDARD: false
           VALIDATE_JSCPD: false
+          VALIDATE_TYPESCRIPT_STANDARD: false
```

Comment:
Disable this validation due to a linting issue. 
Also disabled here https://github.com/actions/typescript-action/commit/7d3890ac9d76ba4390ebca1571f0fd58c883a0b9.
Still not sure why this issue happened